docs/sources/metainfo/component.xml

```diff
@@ -253,17 +253,56 @@
-		</para>
-		<para>
-			The <code>&lt;launchable/&gt;</code> tag has a essential <literal>type</literal> property indicating the system that is used to launch the application. At time, only
-			<code>desktop-id</code> is allowed as value, indicating that an application can be launched via a desktop file. The value of the tag is a
-			<ulink url="https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#desktop-file-id">desktop-file id</ulink> in that case.
-		</para>
-		<para>
-			The <code>&lt;launchable/&gt;</code> tag is allowed to appear multiple times in the metainfo data. In case a software component has multiple launchable entries,
-			the software center might display a dialog to choose which entry to launch. If possible though, it should be avoided to add multiple <literal>launchable</literal> tags.
-		</para>
+		         This optional tag indicates possible methods to launch the software described in this component.
+                         It is allowed to appear multiple times in the metainfo data.
+		</para>
+
+		<para>
+		  The <code>&lt;launchable/&gt;</code> tag has a essential <literal>type</literal> property indicating the system that is used to launch the component.  The following types are allowed:
+                </para>
+                <variablelist>
+                  <varlistentry>
+                    <term>desktop-id</term>
+                    <listitem>
+                      <para>
+                        The application can be launched via a desktop file. The value of the tag is a
+			<ulink url="https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#desktop-file-id">desktop-file id</ulink>.
+                      </para>
+                      <para>
+                        In case a software component has multiple launchable entries,
+			the software center might display a dialog to choose which entry to launch.
+                        If possible though, it should be avoided to add multiple <literal>launchable</literal> tags of type <literal>desktop-id</literal>.
+                      </para>
+                    </listitem>
+                  </varlistentry>
+                  <varlistentry>
+                    <term>service</term>
+                    <listitem>
+                      <para>
+                        The software can be started, stopped, and monitored by the OS "init"
+                        facility, such as systemd. The value of the tag is a name that can be
+                        used with that facility, such as a systemd unit name.
+                      </para>
+                      <para>
+                        If more than one <literal>launchable</literal> tag of type <literal>service</literal> is
+                        present, a UI is expected to let the user manage all these services.  Thus multiple
```

Comment:
Do we need to go into that much detail here?  "Manage" should mean start/stop/restart/enable/disable/monitor, what ever the UI offers for a service.